WFD/swb_import.py

```diff
@@ -95,8 +99,9 @@ def process_all_swb(self, data):
 
         Only increments counter when an swb is updated.
 
-        :param data: dict of all the swb:s in the RBD
+        :param data: list of all the swb:s in the RBD
```

Comment:
This is due to the source data changing one item lists to just the item. Clarifying.